src/include/duckdb/storage/serialization/nodes.json

```diff
@@ -837,6 +838,10 @@
       {"id": 142,
         "name": "multi_file_reader",
         "type": "bool"
+      },
+      {"id": 143,
+        "name": "buffer_size",
```

Comment:
This should have a different name, otherwise the JSON serialization will be confused

Reply:
You can use `"property": "buffer_size"` to keep the variable name in the code the same